### ISSUE TITLE
Choose a different html element for the main wrapper.

### ISSUE THREAD

**Issue by nekpoly** [amytych/react-magic-grid#2]
Hello!
By default `<MagicGrid>` is rendered as a `<div>`. In order to enhance accessibility I am using an `<ul>` with multiple `<li>` items. (Screen readers announce the number of items inside an `<ul>`). Is there a way to replace the default `<div>` with an `<ul>`?

I've already tried the following code. It works, but it also results in a proptype error, since the component expects an array of nodes.

```
<MagicGrid container=".example-class" static animate={false} gutter={0}>
  <ul className="example-class">
    <li>111</li>
    <li>222</li>
  </ul>
</MagicGrid>
```

**Comment by amytych** [comment]
@nekpoly thanks for raising the issue, very good idea to make it more flexible in that regard. Just published v 1.2.1 which should solve your case via `as` prop. Please check the [index.story.js](https://github.com/amytych/react-magic-grid/blob/5b467a0273eb53a7e1fc19a319fe71e790cd424a/index.story.js#L49-L65) for an example of using components different than the default `div`. Let me know if you have any issues 👍

**Comment by nekpoly** [comment]
@amytych thank you for this improvement and your quick response! It worked perfectly! One step closer to a more accessible web! 💯
